src/fparser.f90

```diff
@@ -1015,6 +1017,9 @@ FUNCTION MathConst (str, ibegin, inext, error) RESULT (res)
        IF (str(1:2)=='pi') THEN
           res = 3.14159265358979323846_rn
           in = 3
+       ELSEIF (str(1:2)=='mu') THEN
```

Comment:
interesting solution to this issue. It's a bit weird to change the value of "constants" but the solution does work. One possible danger (for non-phantom sims) is that not all codes choose units where mu=1 (but most do). But happy to merge as-is as it is better than having to change the definition of pmag manually.